server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts

```diff
@@ -296,4 +297,42 @@ investigationInfo.get('/subStatuses/:parentStatus', (request: Request, response:
         });
 });
 
-export default investigationInfo;
\ No newline at end of file
+investigationInfo.post('/updateInvestigationTransfer', (request: Request, response: Response) => {
+    const { epidemiologyNumber } = request.body;
+    logger.info({
+        service: Service.SERVER,
+        severity: Severity.LOW,
+        workflow: 'Investigation transfer',
+        step: `requesting the graphql API to update investigation transfer indication with parameters ${JSON.stringify({
+            epidemiologyNumber
+        })}`,
+        user: response.locals.user.id,
+        investigation: response.locals.epidemiologyNumber
+    })
+    graphqlRequest(UPDATE_INVESTIGATION_TRANSFER, response.locals, {
+        epidemiologyNumber: +epidemiologyNumber
+    })
+        .then((result: any) => {
+            logger.info({
```

Comment:
You should deal with result errors, and in addition send the response also when failing in both them and catch